builtin/game/chat.lua

```diff
@@ -1043,12 +1043,20 @@ core.register_chatcommand("clearobjects", {
 
 		core.log("action", name .. " clears all objects ("
 				.. options.mode .. " mode).")
-		core.chat_send_all("Clearing all objects. This may take a long time."
-				.. " You may experience a timeout. (by "
-				.. name .. ")")
+		if options.mode == "quick" then
+			core.chat_send_all("Clearing all objects (quick mode). (by " .. name .. ")")
+		else
+			core.chat_send_all("Clearing all objects. This may take a long time."
+					.. " You may experience a timeout. (by "
+					.. name .. ")")
+		end
 		core.clear_objects(options)
 		core.log("action", "Object clearing done.")
-		core.chat_send_all("*** Cleared all objects.")
+		if options.mode == "quick" then
+			core.chat_send_all("*** Cleared all objects (quick mode).")
+		else
+			core.chat_send_all("*** Cleared all objects.")
+		end
```

Comment:
The first msg already mentions this, so no need to do it again here IMO.

Reply:
In addition, it's wrong again since not all objects are cleared in quick mode. I'd remove the test for `mode == "quick"` and write something like "Done clearing objects".